designer/client/src/components/toolbars/search/AdvancedSearchOptions.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { TextField, Button, Box, Typography } from "@mui/material";
+
+const transformInput = (input: string, fieldName: string) => {
+    return input === "" ? "" : `${fieldName}:(${input})`;
+};
+
+export function AdvancedSearchOptions({
```

Comment:
There is no option to clear advanced filters. Maybe it's worth adding a button to handle it?